frontend/src/components/BackButton/index.module.css

```diff
@@ -0,0 +1,4 @@
+.backButton {
+  width: fit-content;
+  margin: 24px 0 92px 24px;
```

Comment:
It's not good practice to add layout styles to the component directly since you now can't reuse this `<` as is anywhere else.

I'm okay to leave it in for now, but you can imagine that you would want to extract something like this into a `header` or the likes.

Reply:
Or have a `wrapper` class in the parent like you do for all the other components.